src-i386/RcppExports.cpp

```diff
@@ -0,0 +1,322 @@
+// Generated by using Rcpp::compileAttributes() -> do not edit by hand
```

Comment:
Are we building compiled libraries for i386? If we are, maybe we should ask whether we need to 🙂

Reply:
It relates to my machine, since my laptop is windows. Actually, I am not so sure what is going on here.
A stackoverflow message to explain this https://stackoverflow.com/questions/45507089/src-i386-x64-folders-in-r-package.

Reply:
OK, perfect. We should add `src-i386` and `src-x64` to `.gitignore`, then. The only directory we want which is source-related is `src`.